cmd/convert.go

```diff
@@ -173,7 +177,7 @@ func (x *Convert) Execute(args []string) error {
 	// Update listings with new coin type
 	creationDate := time.Now()
 	var sqliteDB *db.SQLiteDatastore
-	sqliteDB, err = InitializeRepo(repoPath, x.Password, "", x.Testnet, creationDate)
+	sqliteDB, err = InitializeRepo(repoPath, x.Password, "", x.Testnet, creationDate, ct)
```

Comment:
Thinking about this further, this will have to be removed once we have multiple currencies active at once. How are you planning on switching currencies on a per-transaction basis with this design?

Reply:
I'm placated with fixing this later when the multiwallet work is delivered.